docs/user_guide/installation.rst

```diff
@@ -11,6 +11,14 @@ Docker or Singularity.
 
     docker run --rm repronim/neurodocker:latest --help
 
+Note: Some tools require an interactive input during installation (e.g. FSL). Running the container interactively will allow to answer this question:
+
+.. code-block:: bash
+
+    docker run -i --rm
```

Comment:
Maybe better to mention the `--yes` option

Reply:
not sure, how you mean this. Can you include the --yes option somehow in the docker command?

Reply:
ok, I added this now and referred to the examples